packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -84,9 +100,21 @@ class _MyHomePageState extends State<MyHomePage> {
         title: new Text(config.title),
       ),
       body: new Center(
+{{#withPluginHook}}
+        child: new Column(
```

Comment:
Ah ok. I thought this was discussing the `flutter create` case, where we do have a matrix of options.